### PR TITLE
[PUB-2988] Add Campaigns locked state copy for team members after site owner downgrades

### PR DESCRIPTION
**Pull request by albennett**
## Description
Update Campaigns locked state copy for team members after site owner downgrades
https://buffer.atlassian.net/browse/PUB-2988

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/campaigns-list/index.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export default connect(
       isLoading: state.campaignsList.isLoading,
       hasCampaignsFlip: state.organizations.selected?.hasCampaignsFeature,
       ownerEmail: state.organizations.selected?.ownerEmail,
+      shouldDisplayLockedCopy:
+        state.organizations.selected?.planBase === 'pro' &&
```
Comment (hamstu) [inline packages/campaigns-list/index.js:21]: Is it possible that the owner could have downgraded to Free? Might be better to check for `!=` business here instead? 🤔

**Review by hamstu — APPROVED** [review]
Looks great, just left one Q on the logic 😄

File: packages/campaigns-list/index.js (hunk 1 above)
Comment (anafilipadealmeida) [reply packages/campaigns-list/index.js:21]: We have a `locked: orgData.locked` coming from the org parser. It's worth confirming, but that might be enough for us not to have to check plans and roles

Comment (albennett) [reply packages/campaigns-list/index.js:21]: Hey @hamstu free users (or team members of free users) don't see the campaigns option in the sidebar. 

Ooh, thanks @anafilipadealmeida. I originally misunderstood the `locked` meaning, but from looking at the code/testing, it does look like we can use it. 🙌

Comment (hamstu) [reply packages/campaigns-list/index.js]: Good one! Thanks for clarifying @albennett

**Pull request merged**